tests/functional/monitoring/prometheus/metrics/test_ocs_exporter_internal_mode.py

```
@@ -0,0 +1,306 @@
+        actual_namespace,
+    )
+
+
+@pytest.mark.polarion_id("OCS-6013")
+@blue_squad
+@tier1
+@skipif_external_mode
+@skipif_mcg_only
+@skipif_ms_consumer
+@pytest.mark.parametrize(
+    "metric_name,uses_storage_consumer_name",
```

Ruff (0.15.21)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)